api-remove.go

```diff
@@ -195,6 +195,13 @@ func (c Client) RemoveObjectsWithContext(ctx context.Context, bucketName string,
 				contentMD5Base64: sumMD5Base64(removeBytes),
 				contentSHA256Hex: sum256Hex(removeBytes),
 			})
+			if resp != nil {
+				if resp.StatusCode != http.StatusOK {
+					e := httpRespToErrorResponse(resp, bucketName, "")
+					errorCh <- RemoveObjectError{ObjectName: "", Err: e}
```

Comment:
I think we should not send one error and break here.. We need to send the same error multiple times and also continue to the next batch as done in the next code block (if err != nil)

Why ? because some of our users can have their code blocked since here we decided to stop reading from objectsCh channel.

Reply:
ok, good idea. Updated PR.